sqlite_data_norm.c

```diff
@@ -333,6 +338,70 @@ sqlite_fdw_data_norm_bool(sqlite3_context* context, int argc, sqlite3_value** ar
 	sqlite3_result_value(context, arg);
 }
 
+/*
+ * ISO:SQL valid float/double precision values with text affinity such as Infinity or Inf or NaN
+ * will be treated as float like in PostgreSQL console input
+ * Note: SQLite also have Infinity support with real affinity, but this values isn't suitable for insert,
+ * there is any overflow number instead
+ */
+static void
+sqlite_fdw_data_norm_float(sqlite3_context* context, int argc, sqlite3_value** argv)
+{
+	static const char * infs = "Inf";
+	static const char * infl = "Infinity";
+	sqlite3_value* arg = argv[0];
+	int dt = sqlite3_value_type(arg);
+	int l;
+
+	if (dt == SQLITE_FLOAT)
+	{
+		/* The fastest call because expected very often */
+		sqlite3_result_value(context, arg);
+		return;
+	}
+	if (dt != SQLITE3_TEXT && dt != SQLITE_BLOB )
+	{
+		/* INT, NULL*/
+		sqlite3_result_value(context, arg);
+		return;
+	}
+
+	l = sqlite3_value_bytes(arg);
+	if (l > strlen(infl) + 2 || l < strlen(infs))
+	{
+		sqlite3_result_value(context, arg);
+		return;
+	}
+
+	{
```

Comment:
This is variable visibility area only for `minfs`, `minfl`, `pinfs` , `pinfl` and `t`. I think this variables is not for function visibility area.

Reply:
This block does not match with Postgresql coding style.
Could you create new function for this block?  It will be easy to readability and maintenance.

Reply:
This block does not follow Postgres coding style.
Could you create new function for this block? It will be easier to readability and maintenance.

Reply:
Done, please verify, @MinhLA1410

Reply:
Thank you for supporting. It's OK